distributed/protocol/serialize.py

```diff
@@ -572,19 +574,21 @@ class ToPickle:
     to False, the scheduler will raise an exception instead.
     """
 
-    def __init__(self, data):
+    data: T
+
+    def __init__(self, data: T) -> None:
```

Comment:
```suggestion
    def __init__(self, data: T):
```
Return type of `__init__` should always be implicit